addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -291,7 +291,7 @@
             </div>
             <div class='js_progressbar_txt'>
                 <span class="text-success d-none"><i class='fa fa-check'/> File has been uploaded</span>
-                <span class="text-danger d-none"><i class='fa fa-times'/> File could not be saved</span>
+                <span class="text-danger d-none"><p class='fa fa-times pull-left'/> <span>File could not be saved</span></span>
```

Comment:
pull-left toi meme :o That class does not exist anymore in BS4, float-left I guess :wink: Although, a `<p class="fa`? :thinking:

Reply:
No I meant [pull-left](https://github.com/odoo-dev/odoo/blame/f047bb96d40e7e58ae0fbddf0c807763d4362e57/addons/web/static/lib/fontawesome/css/font-awesome.css#L85) but I guess it would be less confusing if I replace it with `float-left` and `mr-1`, done.

Reply:
About the `<p class="fa`, it was a bit dirty but for convenience, as I changed this part of the code 20 times I got a bit lazy at the end.
```xml
<span class="text-danger d-none">
    <p class='fa fa-times float-left mr-1'/>
    <span>File could not be saved</span>
</span>

```
```javascript
$('.text-danger').removeClass('d-none');
$('.text-danger span').text(attachment.error);
```
Anyway since you see everything and noticed it, I wrote it as it should have :roll_eyes: 
```xml
<span class="text-danger d-none">
    <span class='fa fa-times float-left mr-1'/>
    <span class='o_we_super_class'>File could not be saved</span>
</span>

```
```javascript
$('.text-danger').removeClass('d-none');
$('.text-danger .o_we_super_class').text(attachment.error);
```